lib/index.js

```diff
@@ -3,9 +3,41 @@
 exports.register = (server, options, next) => {
 
   server.ext('onPostAuth', (request, reply) => {
-    request.payload.loaded = true;
+    request.plugins['hapi-rate-limit'] = { rate: null };
+
+    if (!(request.route.method === 'get' || request.route.method === 'post' || request.route.method === 'delete')) {
+      return reply.continue();
+    }
+
+    const rate = Object.assign({}, options.defaultRate);
+    const routeSettings = request.route.settings.plugins.rateLimit || null;
+
+    if (routeSettings) {
+      if (routeSettings.enabled === false) {
+        return reply.continue();
+      }
+
+      rate.limit = routeSettings.limit;
+      rate.window = routeSettings.window;
+    }
+
+    const key = `hapi-rate-limit:${request.route.method}:${request.route.path}:${request.auth.credentials.auth.api_key}`;
```

Comment:
I think it's actually [`request.auth.credentials.api_key`](https://github.com/lob/lob-api/blob/master/src/libraries/rateLimitLibrary.js#L36).